### DISCUSSION TITLE
User-defined speeds form 1-X00% fully adjustable

### DISCUSSION THREAD

**Discussion by Arnster** [X1Plus/X1Plus#150] [Ideas]
Hello,
at the moment the speed can only be adjusted in the four levels 50%; 100%;... . I would really appreciate it if the speed could be adjusted freely. For example, like in Klipper.
Many thanks for your work!

**Comment by skydoy** [comment]
Great Ideia 😃

**Comment by W1SPS** [comment]
I would love to see an infinitely variable speed adjustment (Hell I'd be happy with 1 to 11) that also displays the percentage above or below 100% speed. Also I would love to see this set speed set when you send the file to the printer rather than waiting until it's there and THEN adjusting it. I find many times when I could kick the X1C to "Ludicrous" speed I forget. Just let me set my final speed at the beginning and run with it. Or, even better, let me set a starting speed to get good bed adhesion, then after a preset number of layers I can ramp the speed up to a preset amount.
